ui/pages/confirm-send-token/confirm-send-token.container.js

```diff
@@ -18,18 +18,11 @@ const mapStateToProps = (state) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    editTransaction: ({ txData, tokenData, tokenProps: assetDetails }) => {
+    editTransaction: async ({ txData }) => {
       const { id } = txData;
-      dispatch(
-        editTransaction(
-          ASSET_TYPES.TOKEN,
-          id.toString(),
-          tokenData,
-          assetDetails,
-        ),
-      );
-      dispatch(clearConfirmTransaction());
-      dispatch(showSendTokenPage());
+      await dispatch(editExistingTransaction(ASSET_TYPES.TOKEN, id.toString()));
```

Comment:
Out of curiosity, I know this wasn't changed, but what could `id` be that we need to typecast it to string?

Reply:
I honestly have no clue... it should be a string and is a string in the state tree. i'm not sure if there is some higher level area where it is manipulated away from being a JavaScript string object. The .toString() was copied from the confirmation base when the id is manipulated.